DXRVanilla/DeusEx/Classes/Computers.uc

```diff
@@ -103,7 +103,8 @@ state On
         {
             GotoState('Off');
         }
-        else if (VSize(curFrobber.Location - Location) > 150)// DXRando: reduced from 1500 down to 150
+        //else if (VSize(curFrobber.Location - Location) > 150)// DXRando: reduced from 1500 down to 150
+        else if(curFrobber.FrobTarget != self)// DXRando: reduced range from 1500
```

Comment:
well I just found an issue with this lol, May Sung walked in-between me and the computer I was using and I got booted out lmao, which is realistic but annoying, I'll revert this change